Src/BootCamp.Chapter/Algorithm.cs

```diff
@@ -12,8 +12,8 @@ internal class Algorithm
         /// </summary>
         internal static void OptionalTestCases()
         {
-            // int swapCount = 0;
-            // Sort(new int[0], ref swapCount);
+            int swapCount = 3;
+            Sort(new int[] { 1, 3, 2 } , ref swapCount);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4927992)